assets/src/edit-story/components/panels/shapeStyle.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Row, Color } from '../form';
+import { SimplePanel } from './panel';
+import getCommonValue from './utils/getCommonValue';
+
+function ShapeStylePanel({ selectedElements, pushUpdate }) {
+  const backgroundColor = getCommonValue(selectedElements, 'backgroundColor');
+
+  return (
+    <SimplePanel name="style" title={__('Style', 'web-stories')}>
```

Comment:
We need to confirm whether a shape is allowed to be a background per our rules: i.e. can we put a simple shape (rect or star) and click "Use as background" on it. If we do, we should probably disallow "Remove as background" for auto-background-shape. Otherwise it'd simply be recreated immediately.

Confirming further, would we allow isFullbleed customization for the auto-background-shape? If not, does this PR already disable it?